types/bootstrap/js/dist/popover.d.ts

```diff
@@ -122,7 +123,7 @@ declare namespace Popover {
         inserted = "inserted.bs.popover",
     }
 
-    type PopperConfigFunction = (defaultBsPopperConfig: Options) => Partial<Options>;
+    type PopperConfigFunction = (defaultBsPopperConfig: Popper.Options) => Partial<Popper.Options>;
 
     interface Options extends Omit<Tooltip.Options, "popperConfig"> {
```

Comment:
please remove `Omit`
```suggestion
    interface Options extends Tooltip.Options {
```
and remove `popperConfig` and `PopperConfigFunction` from this file